CI seems hitting a regression?

```
=== RUN   TestDockerSuite/TestBuildEmptyEntrypoint
    --- FAIL: TestDockerSuite/TestBuildEmptyEntrypoint (0.19s)
        docker_cli_build_test.go:1709: assertion failed: 
            Command:  /usr/local/cli/docker build -t testbuildentrypoint -
            ExitCode: 1
            Error:    exit status 1
            Stdout:   Sending build context to Docker daemon  2.048kB

            
            Stderr:   Error response from daemon: dockerfile parse error line 2: ENTRYPOINT requires at least one argument
            
            
            Failures:
            ExitCode was 1 expected 0
            Expected no error
```

--- a/vendor.conf
+++ b/vendor.conf
@@ -33,7 +33,7 @@ github.com/imdario/mergo                            1afb36080aec31e0d1528973ebe6
 golang.org/x/sync                                   cd5d95a43a6e21273425c7ae415d3df9ea832eeb
 
 # buildkit
-github.com/moby/buildkit                            fcb87e6b8ccf3631a65799cc56caa76f9117816e # v0.8.0-rc2
+github.com/moby/buildkit                            950603da215ae03b843f3f66fbe86c4876a6f5a1
 github.com/tonistiigi/fsutil                        0834f99b7b85462efb69b4f571a4fa3ca7da5ac9
 github.com/tonistiigi/units                         6950e57a87eaf136bbe44ef2ec8e75b9e3569de2
 github.com/grpc-ecosystem/grpc-opentracing          8e809c8a86450a29b90dcc9efbf062d0fe6d9746

